Project.toml

```diff
@@ -57,4 +59,4 @@ Statistics = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
 Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
 
 [targets]
-test = ["DiffEqCallbacks", "DiffEqDevTools", "DiffEqOperators", "DiffEqProblemLibrary", "SafeTestsets", "SparseArrays", "Statistics", "Test"]
+test = ["DiffEqCallbacks", "DiffEqDevTools", "DiffEqGPU", "DiffEqOperators", "DiffEqProblemLibrary", "SafeTestsets", "SparseArrays", "Statistics", "Test"]
```

Comment:
GPUs should get a different Project.toml that's instantiated. See DiffEqBase

Reply:
@ChrisRackauckas  any idea what is still missing for the GPU tests? Looks like they are not starting..